Formula/qbs.rb

```diff
@@ -1,8 +1,8 @@
 class Qbs < Formula
-  desc "Qt Build Suite"
-  homepage "https://wiki.qt.io/Qt_Build_Suite"
-  url "https://download.qt.io/official_releases/qbs/1.5.0/qbs-src-1.5.0.tar.gz"
-  sha256 "541106d3e53429c5375a58f395413b3cd5a026d91a304a10f36d78b5e39d9085"
+  desc "Qbs"
```

Comment:
We're no longer referring to it as "Qt build suite" for reasons I won't describe here, but I added a fuller description.